requirements/dev.txt

```diff
@@ -290,3 +290,5 @@ setuptools==51.3.1
     #   blockdiag
     #   gunicorn
     #   sphinx
+
+python-on-whales==0.19.0
```

Comment:
This file is managed by `pip-tools` with the source in the neighboring `.in` file. Don't edit it manually. It will be overwritten and this change will be lost.